float32 value
bool wait
---
string message